### DISCUSSION TITLE
Ye, quite simple inni'?

### DISCUSSION THREAD

**Discussion by GWRon** [Blitz-Community/BlitzMax#8] [Worklogs]
Ye, quite simple inni'?

```
pRinT    (   "Heller Werld" )
```

Be sure to upvote this tutorial!

_Originally posted by @Hezkore in https://github.com/Blitz-Community/blitzmax/discussions/3_

**Comment by GWRon** [comment]
Working on improving this old tutorial by @Hezkore

**Reply by Hezkore** [reply]
Can't really improve perfection, eh mate?

**Comment by GWRon** [comment]
think it better should be written
```
print ("Hello World!11!")
```

**Reply by GWRon** [reply]
If you are ok ... convert to issue for easier "fixes # 1234" - or can discussions be closed via commits too? hmm...

btw having the "closes # bla" inside a pull request discussion's "description" keeps it from having to write it in the actual "commits" description - I think.
